### PR TITLE
SEAR-292-Children Sort order

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-292

#### Brief feature description
- sorting ability for children via date of placement

#### Does this PR block/effect other stories?
- n/a

#### Do any commands needs to be run in order to run the project with this PR?
-n/a

#### Code coverage percent diff as a result of this PR?
-n/a

### CODE REVIEW

File: app/javascript/search/common/commonUtils.js [hunk 1]
```diff
@@ -71,6 +71,12 @@ export const checkforDateOrNa = (date) => {
   }
 }
 
+export const sortbyDate = (a, b) => {
```
Comment (ratneshraval) [inline app/javascript/search/common/commonUtils.js]: So why do we need to write our own function for date sorting?

Comment (Nasir2712) [reply app/javascript/search/common/commonUtils.js]: the default sorting was not helping

Comment (ratneshraval) [reply app/javascript/search/common/commonUtils.js]: My guess is the data type is string hence default sorting won't work.
You can tell react-table about the data type or convert it at cell level and hopefully the default sort will work after that.

Comment (Nasir2712) [reply app/javascript/search/common/commonUtils.js]: It works for all strings but doesn't work for the date. Like for example, default sorting works for first name and last name, but not for the date. Tried converting to a date object still does not work.

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Pull request merged**